app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
+        <p class='static-page__text static-page__text--sinai'>Brock 1995 = S.P. Brock. <i>Catalogue of Syriac Fragments (New Finds) in the Library of the Monastery of St Catherine, Mount Sinai</i>(Athens, 1995). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566?referer%3Ddi%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505977880&amp;usg=AOvVaw3YjFUlR-yRhXfx2L9dpFxz"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Gibson = Gibson, Margaret Dunlop. <i>Catalogue of the Arabic Manuscripts in the Convent of S. Catharine on Mount Sinai.</i> Cambridge University Press, 1894. [<%= link_to 'Internet Archive', "https://archive.org/details/catalogueofarabi00sainrich", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Clark =Clark, Kenneth W., ed. <i>Checklist of Manuscripts in St. Catherine&rsquo;s Monastery Mount Sinai. </i>Washington, D.C.: Library of Congress, 1952. [<%= link_to 'Library of Congress', 'https://www.loc.gov/item/52060038&amp;sa=D&amp;source=editors&amp;ust=1683922505978433&amp;usg=AOvVaw11rSYTSzGBp4-g0GroprM9"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Greek New Finds = Nikolopoulos, P. G. Τα νέα ευρήματα του Σινά (Athens, 1998). English translation: A. Hatzopoulos, The New Finds of Sinai (Athens, 1999). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/new-finds-of-sinai/oclc/913489510&referer=brief_results", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Gardthausen = Gardthausen, V. E. <i>Catalogus codicum Graecorum Sinaiticorum</i>. Oxford, 1886. [<%= link_to 'Internet Archive', 'https://archive.org/details/cataloguscodicu00sinagoog&amp;sa=D&amp;source=editors&amp;ust=1683922505978724&amp;usg=AOvVaw3mvUuQiIh3yasFgHnY7jTX"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Kamil = Kāmil, Murād. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai.</i> Harrassowitz, 1970. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2006= G&eacute;hin, Paul. &ldquo;Manuscrits sina&iuml;tiques dispers&eacute;s, I: les fragments arabes et syriaques de Paris.&rdquo; <i>OC</i>90 (2006): 23-43. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505979436&amp;usg=AOvVaw1uYIf9GxuZi6FqmEf8cZfR"', class: 'si-link' %>]</p>
```

Comment:
Missing space: `G&eacute;hin 2006= G&eacute;hin` -> `G&eacute;hin 2006 = G&eacute;hin`

Missing space: `<i>OC</i>90` -> `<i>OC</i> 90`

Reply:
URL should be: https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423